util.inc.sh

```diff
@@ -12,6 +12,11 @@ log_fatal() {
   exit 1
 }
 
+die() {
+  printf "$1\\n" "${@:2}\\n" >&2
+  exit 1
+}
+
```

Comment:
I believe "die" is just a typo for "log_fatal".

Reply:
I see, I can fix that if you like then? I was thinking that might be the case as well. however the message is a bit different though.

Reply:
Sure, I don't think we need two functions with the same purpose. The two callsites with "die" will need to be adjusted. Maybe they can be prefixed with "BUG: " instead of "internal error" similar to the log_fatal callsite in package.inc.sh (line 4).

Reply:
ok. Ill be sending a new PR then(I dont know if it's possible to edit this one by rebasing or something..., if you do, let me know)
regarding the other commit, did you take a look?

Reply:
You can force push to your branch and the pull request will be updated. No need to create a new one.

The other patch looks fine. Thanks for catching that bug.